# Question
Look at this function:

```js
const bailOut = () => {
  for (let i = 0; i < 5; i++) {
    if (i > 3) return;
    console.log('Inside the loop', i);
  }
  console.log('Outside the loop');
};
```

Is that `console.log('Outside the loop');` ever going to run? Why or why not?

# Answer

The `console.log('Outside the loop')` is not going to run as long as there is a return statement within the loop. The return statement, after the speific condition in the if statement is met, will terminate the entire function so anything written after the loop will be ignored.

# Teacher Feedback
